packages/block-library/src/quote/block.json

```diff
@@ -7,18 +7,10 @@
 	"keywords": [ "blockquote", "cite" ],
 	"textdomain": "default",
 	"attributes": {
-		"value": {
-			"type": "string",
-			"source": "html",
-			"selector": "blockquote",
-			"multiline": "p",
-			"default": "",
-			"__experimentalRole": "content"
-		},
 		"citation": {
```

Comment:
Perhaps it would be prudent to rename this attribute to `caption` now? Otherwise it may be a bit misleading, as well as confusing if we ever add a block attribute for the `<blockquote>` element's `cite` HTML attribute.

Reply:
I would imagine we wouldn't want to avoid supporting the `cite` attribute given that it seems like it would be easy to misuse (I don't understand why, but the spec says it's not meant for people, mostly meant for servers, and I think it's more of an artifact than something we should support today).

that being said there's enough confusion in this PR about what `<cite>` is and isn't so I'm happy to find a better name. the HTML5 spec uses "attribution" to describe the content inside the `<figcapture>` and that would make sense to me and clearly distinguish the attribution from one or more citations, which can legitimately fall inside the quote as well as inside the attribution.

how do you feel about `attribution`?

Reply:
`attribution` sounds good to me.